easybuild/easyconfigs/o/ORCA/ORCA-4.2.0-OpenMPI-3.1.4.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = "PackedBinary"
+
+name = "ORCA"
+version = '4.2.0'
+
+ompi_ver = '3.1.4'
+versionsuffix = '-OpenMPI-%s' % ompi_ver
+
+homepage = 'https://orcaforum.kofo.mpg.de'
+description = """ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum chemistry
+ with specific emphasis on spectroscopic properties of open-shell molecules.
+  It features a wide variety of standard quantum chemical methods ranging from semiempirical methods to DFT to single-
+ and multireference correlated ab initio methods.
+ It can also treat environmental and relativistic effects."""
+
+toolchain = {'name': 'dummy', 'version': ''}
```

Comment:
@lexming As discussed in the Slack channel, please use `GCC` or `gompi` as toolchain instead, so the ORCA installation also works when using a hierarchical module naming scheme.

`gompi` may make more sense (since then you can drop the `versionsuffix` too)

Reply:
Done :+1: